### ISSUE TITLE
Unable to Login into SocioBoard

### ISSUE THREAD

**Issue by yogesh23021995** [socioboard/Socioboard-5.0#159]
Hi Team,

When I am trying to login, I am getting below error.

![IMG-20200406-WA0009](https://user-images.githubusercontent.com/50060627/78751493-fa02e780-798f-11ea-9024-2a725012a236.jpg)

**Comment by sureshbabu1995** [comment]
Hi, @yogesh23021995 

I hope the interactive demo cleared this issue ...

**Comment by yogesh23021995** [comment]
Yes, Thank you so much for resolving the issue, but still I am not able to add any facebook, can you please help me resolve the issue, I am getting URL blocked error even after adding the account.

**Comment by sureshbabu1995** [comment]
Hi, @yogesh23021995 
Please check the Redirect URL which you've added in the Facebook application.

**Comment by hcortez-bit** [comment]
Hi @yogesh23021995 
i wanted to see if you found a solution to socioboard 4 with regards to the microservices and social logins.
any help would be appreciated
